modules/ztp-about-ztp-and-factory-install-clusters.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/ztp-factory-install-clusters.adoc
+
+[id="about-ztp-and-factory-installing-clusters_{context}"]
+= About ZTP and factory installing cluster
+
+You can install clusters at scale with {rh-rhacm-first} (ACM) using the assisted installer (AI) in a factory setting. The spoke cluster installation is done using zero touch provisioning (ZTP) in a disconnected environment.
+
+ACM manages clusters in a hub and spoke architecture, where a single hub cluster manages many spoke clusters. Hub clusters running ACM provision and deploy the spoke clusters using ZTP and AI.
+
+The AI service handles provisioning of {product-title} of the spoke clusters (single node Openshift or compact clusters) running on bare metal. ACM ships with and deploys the assisted installer when the `MultiClusterHub` custom resource is installed.
+
+With ZTP and AI, you can provision {product-title} multiple individual machines in a factory-type or partner setting at scale. A high level overview of ZTP in a factory-type environment is as follows:
+
+* A hub cluster running ACM manages a disconnected internal registry that mirrors the {product-title} release images. The internal registry is used to provision the spoke cluster.
```

Comment:
A hub cluster running RHACM

Reply:
My bad but this file is not used in final copy, but I still had it incorrect in modules/ztp-intro-factory-install.adoc which I just corrected https://deploy-preview-40955--osdocs.netlify.app/openshift-enterprise/latest/scalability_and_performance/ztp-factory-install-clusters.html#introduction-factory-install_pipeline_ztp-factory-install